packages/ng/skeleton/skeleton-index-table/skeleton-index-table.component.html

```diff
@@ -0,0 +1,22 @@
+<table class="indexTable mod-layoutFixed skeleton is-loading" [class.mod-dark]="dark" inert="inert">
+	<thead class="indexTable-head">
+		<tr class="indexTable-head-row">
+			@for (col of colsNumber(); track col) {
+			<th class="indexTable-head-row-cell" scope="col">
+				<span class="skeleton-item" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
+			</th>
+			}
+		</tr>
+	</thead>
+	<tbody class="indexTable-body">
+		@for (row of rowsNumber(); track row) {
+		<tr class="indexTable-body-row">
+			@for (col of colsNumber(); track col) {
+			<td class="indexTable-body-row-cell">
+				<span class="skeleton-item" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
+			</td>
+			}
+		</tr>
+		}
+	</tbody>
+</table>
```

Comment:
In general, should we use `aria-live` on loading content?

For orientation, is it better to have a separate markup made up of divs, which works as well visually and which semantically says there is no content, or an empty table which can be announced, but without any hint about what is inside when loading?

Reply:
If I'm to believe [Adrian Roselli](https://adrianroselli.com/2020/11/more-accessible-skeletons.html) an `aria-busy` would only be understood by Jaws. Here the `inert` covers both the `aria-hidden` that he recommends as a complement, and at the same time deactivates the semantics.